content/operate/rs/clusters/remove-node.md

```diff
@@ -9,20 +9,18 @@ description: Remove a node from your Redis Enterprise cluster.
 linkTitle: Remove node
 weight: 80
 ---
-There are various reasons why you may want to remove a node in Redis
-Enterprise Software:
+You might want to remove a node from a Redis Enterprise cluster for one of the following reasons:
 
-- You no longer need the extra capacity, meaning you want to permanently remove the node.
-- You would like to replace a faulty node with a healthy node.
-- You would like to replace a healthy node with a different node.
+- You no longer need the extra capacity, so you want to [permanently remove the node](#permanently-remove-a-node).
```

Comment:
Does the different sentence structure matter here? ("You want to permanently remove..." on the first and then "To replace..." on the others)? Maybe add "You want..." at the start or rewrite the first as "To permanently remove the node because you no longer need the extra capacity"? Purely a suggestion; not sure it makes much difference :-)